### ISSUE TITLE
Cannot find module 'eslint-config-strict-standard'

### ISSUE THREAD

**Issue by zeke** [denis-sokolov/strict-standard#7]
There's a strict mode conversation going on at https://github.com/electron/electron/pull/5259

I came across https://github.com/feross/standard/issues/382 which led me here.

I'm getting an error when I try to run `strict-standard`:

```
~/electron/electron remove-strict-mode*
❯ npm ls standard                    
electron@0.37.7 /Users/zeke/electron/electron
└── standard@6.0.8 


~/electron/electron remove-strict-mode*
❯ npm ls strict-standard      
electron@0.37.7 /Users/zeke/electron/electron
└── strict-standard@2.0.3 


~/electron/electron remove-strict-mode*
❯ ./node_modules/.bin/standard

~/electron/electron remove-strict-mode*
❯ ./node_modules/.bin/strict-standard 
strict-standard: Unexpected linter output:

Error: Cannot find module 'eslint-config-strict-standard' from '/Users/zeke/electron/electron/node_modules/strict-standard'
Referenced from: /Users/zeke/electron/electron/node_modules/strict-standard/.eslintrc
    at Function.module.exports [as sync] (/Users/zeke/electron/electron/node_modules/resolve/lib/sync.js:33:11)
    at resolve (/Users/zeke/electron/electron/node_modules/eslint/lib/config/config-file.js:416:38)
    at load (/Users/zeke/electron/electron/node_modules/eslint/lib/config/config-file.js:435:24)
    at /Users/zeke/electron/electron/node_modules/eslint/lib/config/config-file.js:355:36
    at Array.reduceRight (native)
    at applyExtends (/Users/zeke/electron/electron/node_modules/eslint/lib/config/config-file.js:338:28)
    at Object.load (/Users/zeke/electron/electron/node_modules/eslint/lib/config/config-file.js:462:22)
    at loadConfig (/Users/zeke/electron/electron/node_modules/eslint/lib/config.js:67:33)
    at new Config (/Users/zeke/electron/electron/node_modules/eslint/lib/config.js:193:38)
    at CLIEngine.executeOnFiles (/Users/zeke/electron/electron/node_modules/eslint/lib/cli-engine.js:493:28)

If you think this is a bug in `strict-standard`, open an issue: https://github.com/denis-sokolov/strict-standard/issues
```

**Comment by denis-sokolov** [comment]
Fixed in 2.0.4, thank you.

**Comment by zeke** [comment]
Thanks! Hit another snag with 2.0.4: #8
